spec/unit/provider/user/solaris_spec.rb

```diff
@@ -31,15 +36,6 @@
     p
   end
 
-  supported_useradd_options = {
-    'comment' => "-c",
-    'gid' => "-g",
-    'uid' => "-u",
-    'shell' => "-s"
-  }
-
-  include_examples "a useradd-based user provider", supported_useradd_options
```

Comment:
Were the other tests passing? it might be worth separating the three lock related describe blocks out of ```spec/support/shared/unit/provider/useradd_based_user_provider.rb``` into another file like ```usermod_based_lock_user_provider``` and then adding a similar include_examples to the other tests that include this file.